deploy/clowdapp.yml

```
@@ -1381,6 +1387,8 @@ parameters:
   value: host-inventory
 - name: PATH_PREFIX
   value: api
+- name: CONSOLEDOT_HOSTNAME
+  value: 'console.redhat.com'
```

Shouldn't me localhost as well here? 
This must be overwritten in stage/prod deployment files

```suggestion
  value: 'localhost'
```

---

Oh, for some reason I thought it was already "localhost". Yes I agree that it should be